analyzer/tools/build-logger/src/ldlogger-hooks.c

```diff
@@ -46,19 +46,12 @@
     the program to execute, so we're not reusing CC_LOGGER_CALL_EXEC.          \
   */                                                                           \
   tryLog(path, argv);                                                          \
-  /*                                                                           \
-    Note that envp is not passed through (which would be the environment that  \
-    that the process will be spawned it), but is replaced by environ, which is \
-    a global variable that stores our environment. We added quite a few        \
-    variables into it, and for some reason, they are not present in envp.      \
-  */                                                                           \
```

Comment:
It was a while since I wrote this, but I recall that some codechecker (`CC_*`) and logger related variables were not present in envp, but should've been. Can you verify thats not an issue?

Reply:
I'm not sure, which environment variable was involved in your case, but this function call blocked the forwarding of all environment variable. I assume, this `cc_` was among them too.